core/block/simple/dataview/dataview.go

```diff
@@ -520,3 +522,29 @@ func (s *Dataview) UpdateRelationOld(relationKey string, rel model.Relation) err
 func (d *Dataview) IsEmpty() bool {
 	return d.content.TargetObjectId == "" && len(d.content.Views) == 0
 }
+
+func (d *Dataview) HasEmptyContent() bool {
+	if d.content == nil {
+		return true
+	}
+
+	var view *model.BlockContentDataviewView
+	switch len(d.content.Views) {
+	case 0:
+		return true
+	case 1:
+		view = d.content.Views[0]
+	default:
+		return false
+	}
+
+	if view.DefaultObjectTypeId != "" ||
+		view.DefaultTemplateId != "" ||
+		len(view.Filters) != 0 ||
+		len(view.Sorts) > 1 ||
```

Comment:
Why do we check `len(view.Sorts)` as more then 1? Just want to understand the logic 😃

Reply:
On creation of new dataview block we add sorting by lastModified by default:
https://github.com/anyproto/anytype-heart/blob/main/core/block/editor/dataview/dataview.go#L528
so if we find some other sorts in the view, we can consider dataview object as modified